exporters/write_buffers/base_buffer.py

```diff
@@ -118,7 +118,8 @@ class GroupingBufferFilesTracker(object):
     that is cleaned up when calling close().
     """
 
-    def __init__(self, formatter, compression_format):
+    def __init__(self, formatter=None, compression_format=None, *args, **kwargs):
```

Comment:
which is the point of having *args after two parameters with default values? In order args to actually have some value, you must always pass some value to formatter and to compression_format. So, either formatter and compression_format shouldn't have default values, or *args shouldn't be there.